### PR TITLE
RET-927,928 and checkstyle

### PR DESCRIPTION
**Pull request by hsjhita**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-927
https://tools.hmcts.net/jira/browse/RET-928

### Change description ###

Fixed CaseSource reporting office

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java [hunk 1]
```diff
@@ -13,22 +15,34 @@
...
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.MIGRATION_CASE_SOURCE;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.SCOTLAND_LISTING_CASE_TYPE_ID;
 
 public class CaseSourceLocalReportTest {
+
+    private ListingDetails listingDetails;
+    private ListingData listingData;
+    private List<SubmitEvent> submitEvents;
+
+    CaseSourceLocalReport caseSourceLocalReport;
```
Comment (jthmcts) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java:32]: This ought to be private like the rest

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java [hunk 2]
```diff
@@ -13,22 +15,34 @@
...
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.ENGLANDWALES_LISTING_CASE_TYPE_ID;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.ET1_ONLINE_CASE_SOURCE;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.FLAG_ECC;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.MANUALLY_CREATED_POSITION;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.MIGRATION_CASE_SOURCE;
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.SCOTLAND_LISTING_CASE_TYPE_ID;
 
 public class CaseSourceLocalReportTest {
```
Comment (jthmcts) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java:26]: New tests should be using JUnit 5

**Review by jthmcts — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java (hunk 1 above)
Comment (hsjhita) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java]: Refactored

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java (hunk 2 above)
Comment (hsjhita) [reply src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/CaseSourceLocalReportTest.java]: Refactored class to use Junit 5

**Pull request merged**